src/main/webapp/resources/js/components/samples/components/SampleMetadata.tsx

```diff
@@ -12,12 +12,12 @@ import {
 import { AddNewMetadata } from "./AddNewMetadata";
 import { useRemoveSampleMetadataMutation } from "../../../apis/samples/samples";
 import { ContentLoading } from "../../loader";
-import { IconEdit, IconPlusCircle, IconRemove } from "../../icons/Icons";
+import { IconPlusCircle } from "../../icons/Icons";
 import { MetadataRolesProvider } from "../../../contexts/metadata-roles-context";
 import { EditMetadata } from "./EditMetadata";
 import AutoSizer from "react-virtualized-auto-sizer";
 import { FixedSizeList as VList } from "react-window";
```

Comment:
You will probably need to add the type definitions for these: `https://www.npmjs.com/package/@types/react-window`

Reply:
These have already been added